Registry: Global search (#1043)
wandb/docs: content/guides/models/registry/search_registry.md

Reviewer: Can we add a gif reflecting the search workflow?

---
menu:
  default:
    identifier: search_registry
    parent: registry
title: Find registry items 
weight: 7
---

> You can browse the W&B Registry App interactively to find a specific collection, registry, artifact version, or collection tag. However, this method can be time-consuming when a registry has a large number of items.

Reviewer: ```suggestion
You can browse the W&B Registry App interactively to find a specific collection, registry, artifact version, or collection tag. 
```
Hmm I don't think this is the right place to clarify this.

> As you add more artifacts to the W&B Registry App, you may need to search for a specific registry, collection, artifact version tag, or collection tag. You can click through the W&B Registry App to find a specific collection, registry, artifact version, or collection tag. However, this method can be time-consuming as the number of artifacts in your organization increases. 

Reviewer 2: ```suggestion
You can browse the W&B Registry App interactively to find a specific collection, registry, artifact version, or collection tag. However, this method can be time-consuming when a registry has a large number of items.
```

Use the global search bar to quickly find a specific registry, collection, artifact version tag, or collection tag efficiently. To search globally:

1. Navigate to the W&B Registry App at https://wandb.ai/registry/. 

> 2. Enter the search term in the search bar at the top of the page. 

Reviewer: the really important thing to call out is that you must hit enter - this isn't a "load results as you type" interface

> Instead of manually clicking through a registry or collection, you can use the global search bar to quickly find a specific registry, collection, artifact version tag, or collection tag. To search globally:

Reviewer 2: ```suggestion
Use the global search bar to find a specific registry, collection, artifact version tag, or collection tag efficiently. To search globally:
```

> Search results appear below the search bar if the term you specify matches an existing registry, collection, artifact version tag, or collection tag.

Reviewer: ```suggestion
Search results appear below the search bar if the term you specify matches an existing registry, collection name, artifact version tag, collection tag, or alias.
```

> Only registries, collections, artifact version tags, and collection tags that you have permission to view appear in the search results.

Reviewer 2: ```suggestion
Only items that you have permission to view appear in the search results.
```

---
menu:
  default:
    identifier: search_registry
    parent: registry
title: Find registry items 
weight: 7
---

You can browse the W&B Registry App interactively to find a specific collection, registry, artifact version, or collection tag.

> Use the global search bar to quickly find a specific registry, collection, artifact version tag, or collection tag efficiently. To search globally:

Reviewer 3: should we mention alias here as well?
